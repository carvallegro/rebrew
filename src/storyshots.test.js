import initStoryshots, {
  snapshotWithOptions
} from '@storybook/addon-storyshots'

initStoryshots({
  test: snapshotWithOptions(story => ({
    createNodeMock: element => {
      // element.setAttribute = jest.fn()
      // element.removeAttribute = jest.fn()
      // return element
      return document.createElement('div')
    }
  }))
})
