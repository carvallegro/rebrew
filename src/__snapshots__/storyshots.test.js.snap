// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Application Layout Default Render 1`] = `
<div
  className="css-1s71d7b"
>
  <div
    className="css-cfo52d"
  >
    <header
      className="css-1hfnl0x"
    >
      <button
        className="css-5sb8u8"
        onClick={[Function]}
        type="button"
      >
        <span>
          <svg
            aria-hidden="true"
            className="css-n86my2"
            height="1em"
            role="img"
            viewBox="0 0 24 24"
            width="1em"
          >
            <defs>
              <path
                d="M1 10h14c.667 0 1 .332 1 1 0 .62-.287.952-.862.995L15 12H1c-.667 0-1-.333-1-1 0-.619.287-.95.862-.995L1 10h14zm14-5c.667 0 1 .333 1 1s-.333 1-1 1H1c-.667 0-1-.334-1-1.002C0 5.329.333 4.997 1 5zM1 0h14c.667 0 1 .333 1 1 0 .619-.287.95-.862.995L15 2H1c-.667 0-1-.333-1-1C0 .381.287.05.862.005L1 0h14z"
                id="hamburguer-menu_svg__a"
              />
            </defs>
            <use
              transform="translate(4 6)"
              xlinkHref="#hamburguer-menu_svg__a"
            />
          </svg>
        </span>
      </button>
      <a
        className="css-1iesxmr"
        onClick={[Function]}
      >
        <svg
          className="css-y9zn78"
          height="42px"
          width="45px"
        >
          logo.svg
        </svg>
      </a>
      <button
        className="css-1y4or5a"
        disabled={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
        title="Upload content"
        type="button"
      >
        <span>
          <svg
            aria-hidden="true"
            className="css-n86my2"
            height="1em"
            role="img"
            viewBox="0 0 24 24"
            width="1em"
          >
            <path
              d="M19 18c.667-.004 1 .33 1 1.001 0 .672-.333 1.005-1 .999H5.002c-.67 0-1.004-.334-1.002-1 .002-.667.335-1 1-1zM12.707 4.293l5.001 5.001a1 1 0 01-1.414 1.414L13 7.414V15c0 .667-.333 1-1 1s-1-.333-1-1V7.414l-3.293 3.293a1 1 0 11-1.414-1.414l5-5a1 1 0 011.414 0z"
            />
          </svg>
        </span>
      </button>
    </header>
    <main
      className="css-ycgarc"
    />
  </div>
</div>
`;

exports[`Storyshots Backgrounds Table Default Render 1`] = `
<table
  className="css-1iy4mqq"
>
  <thead
    className="css-0"
  >
    <tr
      className="css-0"
    >
      <th
        className="css-124yh7v"
        data-name="name"
      >
        <span
          className="css-1vben6a"
          onClick={[Function]}
        >
          Name
          <svg
            className="css-1uir35z"
            height="1em"
            viewBox="0 0 24 24"
            width="1em"
          >
            <polygon
              className="css-193cgrs"
              points="12 4.94 16.95 9.89 18.36 8.47 12 2.11 5.64 8.47 7.05 9.89 12 4.94"
            />
            <polygon
              className="css-0"
              points="12 19.06 7.05 14.11 5.64 15.53 12 21.89 18.36 15.53 16.95 14.11 12 19.06"
            />
          </svg>
        </span>
      </th>
      <th
        className="css-1366b59"
        data-name="proficiencies"
      >
        Proficiencies
      </th>
      <th
        className="css-83rr42"
        data-name="feature"
      >
        Features
      </th>
    </tr>
  </thead>
  <tbody
    className="css-0"
  >
    <tr
      className="css-0"
      onClick={null}
    >
      <td
        className="css-tiri1u"
      >
        Acolyte
      </td>
      <td
        className="css-jasaoa"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Gain all of
          </div>
          <div
            className="css-1pnujgh"
          >
            Insight, Religion
          </div>
        </div>
      </td>
      <td
        className="css-14yspz7"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Shelter of the Faithful
          </div>
          <div
            className="css-1pnujgh"
          >
            <div
              className="css-16ewyma"
            >
              <p
                className="css-1ocwri7"
              >
                As an acolyte, you command the respect of those who share your faith, and you can perform the religious ceremonies of your deity. You and your adventuring companions can expect to receive free healing and care at a temple, shrine, or other established presence of your faith, though you must provide any material components needed for spells. Those who share your religion will support you (but only you) at a modest lifestyle.
              </p>
              <p
                className="css-1ocwri7"
              >
                You might also have ties to a specific temple dedicated to your chosen deity or pantheon, and you have a residence there. This could be the temple where you used to serve, if you remain on good terms with it, or a temple where you have found a new home. While near your temple, you can call upon the priests for assistance, provided the assistance you ask for is not hazardous and you remain in good standing with your temple.
              </p>
            </div>
          </div>
        </div>
      </td>
    </tr>
    <tr
      className="css-0"
      onClick={null}
    >
      <td
        className="css-tiri1u"
      >
        Criminal / Spy
      </td>
      <td
        className="css-jasaoa"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Gain all of
          </div>
          <div
            className="css-1pnujgh"
          >
            Deception, Stealth
          </div>
        </div>
      </td>
      <td
        className="css-14yspz7"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Criminal Contact
          </div>
          <div
            className="css-1pnujgh"
          >
            <div
              className="css-16ewyma"
            >
              <p
                className="css-1ocwri7"
              >
                Who acts like liaison of you to any network of criminals, you have trustworthy and reliable contact from them. How to get the messages from your contact and also to your contact is well known to you. Even over great distances too you can get the messages; specially you know the corrupt caravan masters and also the local messengers even the seedy sailors who can able to deliver messages for you.
              </p>
            </div>
          </div>
        </div>
      </td>
    </tr>
    <tr
      className="css-0"
      onClick={null}
    >
      <td
        className="css-tiri1u"
      >
        Folk Hero
      </td>
      <td
        className="css-jasaoa"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Gain all of
          </div>
          <div
            className="css-1pnujgh"
          >
            Animal Handling, Survival
          </div>
        </div>
      </td>
      <td
        className="css-14yspz7"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Rustic Hospitality
          </div>
          <div
            className="css-1pnujgh"
          >
            <div
              className="css-16ewyma"
            >
              <p
                className="css-1ocwri7"
              >
                You have come from a humble social rank, but you could destined for so much more. Of course already the peoples those who are your home village regard you like their champion and also your destiny do call you to stand against the tyrants and also monsters those are threaten the common folk everywhere.
              </p>
            </div>
          </div>
        </div>
      </td>
    </tr>
    <tr
      className="css-0"
      onClick={null}
    >
      <td
        className="css-tiri1u"
      >
        Noble
      </td>
      <td
        className="css-jasaoa"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Gain all of
          </div>
          <div
            className="css-1pnujgh"
          >
            History, Persuasion
          </div>
        </div>
      </td>
      <td
        className="css-14yspz7"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Position of Privilege
          </div>
          <div
            className="css-1pnujgh"
          >
            <div
              className="css-16ewyma"
            >
              <p
                className="css-1ocwri7"
              >
                Actually, people are inclined to think the best of you, so you should thanks to your noble birth. The advantages of this feature are
              </p>
              <p
                className="css-1ocwri7"
              >
                
              </p>
              <p
                className="css-1ocwri7"
              >
                You’re always welcome in the high society and also all the people do start to assume that you have the right to be wherever you are.
              </p>
              <p
                className="css-1ocwri7"
              >
                But the common folk and the merchants make each and every effort to accommodate you and also avoid the displeasure of your’s
              </p>
              <p
                className="css-1ocwri7"
              >
                The other peoples who has high birth they treat you as a member of the same social sphere. If you need to secure an audience, you can secure with the local noble.
              </p>
              <p
                className="css-1ocwri7"
              >
                
              </p>
            </div>
          </div>
        </div>
      </td>
    </tr>
    <tr
      className="css-0"
      onClick={null}
    >
      <td
        className="css-tiri1u"
      >
        Sage
      </td>
      <td
        className="css-jasaoa"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Gain all of
          </div>
          <div
            className="css-1pnujgh"
          >
            Arcana, History
          </div>
        </div>
      </td>
      <td
        className="css-14yspz7"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Researcher
          </div>
          <div
            className="css-1pnujgh"
          >
            <div
              className="css-16ewyma"
            >
              <p
                className="css-1ocwri7"
              >
                You know more oftenly like where and from whom you can able to obtain the information, that whenever you attempt to learn or recall the piece of lore, if you don’t have any idea about that specific information.
              </p>
              <p
                className="css-1ocwri7"
              >
                
              </p>
              <p
                className="css-1ocwri7"
              >
                From the other learned creature or person, from any library, from the scriptorium, from the university, this information usually comes.
              </p>
              <p
                className="css-1ocwri7"
              >
                
              </p>
              <p
                className="css-1ocwri7"
              >
                Here your Dungeon Master might rule the knowledge which you seek is secreted away in an almost inaccessible place or simply it can not be found.
              </p>
              <p
                className="css-1ocwri7"
              >
                
              </p>
              <p
                className="css-1ocwri7"
              >
                Unearthing the deepest secrets of the multiverse able to require an adventure or even the whole campaign.
              </p>
            </div>
          </div>
        </div>
      </td>
    </tr>
    <tr
      className="css-0"
      onClick={null}
    >
      <td
        className="css-tiri1u"
      >
        Soldier
      </td>
      <td
        className="css-jasaoa"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Gain all of
          </div>
          <div
            className="css-1pnujgh"
          >
            Athletics, Intimidation
          </div>
        </div>
      </td>
      <td
        className="css-14yspz7"
      >
        <div
          className="css-1sq1gba"
        >
          <div
            className="css-in3yi3"
          >
            Military Rank
          </div>
          <div
            className="css-1pnujgh"
          />
        </div>
      </td>
    </tr>
  </tbody>
</table>
`;

exports[`Storyshots Header Default Render 1`] = `
<header
  className="css-1hfnl0x"
>
  <button
    className="css-5sb8u8"
    onClick={[Function]}
    type="button"
  >
    <span>
      <svg
        aria-hidden="true"
        className="css-n86my2"
        height="1em"
        role="img"
        viewBox="0 0 24 24"
        width="1em"
      >
        <defs>
          <path
            d="M1 10h14c.667 0 1 .332 1 1 0 .62-.287.952-.862.995L15 12H1c-.667 0-1-.333-1-1 0-.619.287-.95.862-.995L1 10h14zm14-5c.667 0 1 .333 1 1s-.333 1-1 1H1c-.667 0-1-.334-1-1.002C0 5.329.333 4.997 1 5zM1 0h14c.667 0 1 .333 1 1 0 .619-.287.95-.862.995L15 2H1c-.667 0-1-.333-1-1C0 .381.287.05.862.005L1 0h14z"
            id="hamburguer-menu_svg__a"
          />
        </defs>
        <use
          transform="translate(4 6)"
          xlinkHref="#hamburguer-menu_svg__a"
        />
      </svg>
    </span>
  </button>
  <a
    className="css-1iesxmr"
    onClick={[Function]}
  >
    <svg
      className="css-y9zn78"
      height="42px"
      width="45px"
    >
      logo.svg
    </svg>
  </a>
  <button
    className="css-1y4or5a"
    disabled={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onMouseEnter={[Function]}
    title="Upload content"
    type="button"
  >
    <span>
      <svg
        aria-hidden="true"
        className="css-n86my2"
        height="1em"
        role="img"
        viewBox="0 0 24 24"
        width="1em"
      >
        <path
          d="M19 18c.667-.004 1 .33 1 1.001 0 .672-.333 1.005-1 .999H5.002c-.67 0-1.004-.334-1.002-1 .002-.667.335-1 1-1zM12.707 4.293l5.001 5.001a1 1 0 01-1.414 1.414L13 7.414V15c0 .667-.333 1-1 1s-1-.333-1-1V7.414l-3.293 3.293a1 1 0 11-1.414-1.414l5-5a1 1 0 011.414 0z"
        />
      </svg>
    </span>
  </button>
</header>
`;

exports[`Storyshots HomepageLink Default Render 1`] = `
<div
  className="css-goojuo"
  elevation={0}
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <p
    className="css-1pha4dg"
  >
    Link Text
  </p>
  <svg
    aria-hidden="true"
    className="css-n86my2"
    height="1em"
    role="img"
    viewBox="0 0 24 24"
    width="1em"
  >
    <path
      d="M19.707 12.707l-5.001 5.001a1 1 0 11-1.414-1.414L16.586 13H5c-.667 0-1-.332-1-1 0-.668.333-1 1-1h11.586l-3.293-3.293a1 1 0 111.414-1.414l5 5a1 1 0 010 1.414z"
    />
  </svg>
</div>
`;

exports[`Storyshots HomepageLink When Disabled 1`] = `
<div
  className="css-14occkn"
  elevation={0}
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <p
    className="css-1pha4dg"
  >
    Link Text
  </p>
  <svg
    aria-hidden="true"
    className="css-n86my2"
    height="1em"
    role="img"
    viewBox="0 0 24 24"
    width="1em"
  >
    <path
      d="M19.707 12.707l-5.001 5.001a1 1 0 11-1.414-1.414L16.586 13H5c-.667 0-1-.332-1-1 0-.668.333-1 1-1h11.586l-3.293-3.293a1 1 0 111.414-1.414l5 5a1 1 0 010 1.414z"
    />
  </svg>
</div>
`;

exports[`Storyshots Logo Custom Size Render 1`] = `
<svg
  className="css-1a4qcxd"
  height="100px"
  width="100px"
>
  logo.svg
</svg>
`;

exports[`Storyshots Logo Default Render 1`] = `
<svg
  className="css-y9zn78"
  height="42px"
  width="45px"
>
  logo.svg
</svg>
`;

exports[`Storyshots Spell Fields/CastingTime Default Render 1`] = `
<div
  className="css-0"
>
  <label
    className="css-1tdx2hb"
    htmlFor="casting_time"
  >
    Casting time
    <span
      className="css-fpulsy"
      title="Required field"
    >
      *
    </span>
  </label>
  <div
    className="css-o1mvac"
  >
    <input
      className="css-15ndqf0"
      id="casting_time"
      name="casting_time"
      placeholder="Reaction, 1 action..."
      required={true}
      type="text"
    />
  </div>
</div>
`;

exports[`Storyshots Spell Fields/Components Default Render 1`] = `
<div
  className="css-0"
>
  <label
    className="css-1tdx2hb"
    htmlFor="spellComponents"
  >
    Components
    <span
      className="css-fpulsy"
      title="Required field"
    >
      *
    </span>
  </label>
  <div
    className="css-0"
  >
    <div
      aria-expanded={false}
      aria-haspopup="listbox"
      aria-labelledby="downshift-0-label"
      aria-owns={null}
      role="combobox"
    >
      <div
        className="css-7bq0j0"
        onClick={[Function]}
      >
        <input
          aria-activedescendant={null}
          aria-autocomplete="list"
          aria-controls={null}
          autoComplete="off"
          className="css-1hneurt"
          id="downshift-0-input"
          name="components"
          onBlur={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          placeholder="Choose amongst V, M and S"
          required={true}
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Spell Fields/Description Default Render 1`] = `
<div
  className="css-0"
>
  <label
    className="css-1tdx2hb"
    htmlFor="text"
  >
    Wrapper
    <span
      className="css-fpulsy"
      title="Required field"
    >
      *
    </span>
  </label>
  <textarea
    className="css-15ndqf0"
    id="text"
    name="text"
    placeholder="Enter the description"
    required={true}
  />
</div>
`;

exports[`Storyshots Spell Fields/Name Default Render 1`] = `
<div
  className="css-0"
>
  <label
    className="css-1tdx2hb"
    htmlFor="name"
  >
    Name
    <span
      className="css-fpulsy"
      title="Required field"
    >
      *
    </span>
  </label>
  <div
    className="css-o1mvac"
  >
    <input
      className="css-15ndqf0"
      id="name"
      name="name"
      placeholder="Enter the spell name"
      required={true}
      type="text"
    />
  </div>
</div>
`;

exports[`Storyshots Spell Fields/School Default Render 1`] = `
<div
  className="css-0"
>
  <label
    className="css-1tdx2hb"
    htmlFor="school"
  >
    School
    <span
      className="css-fpulsy"
      title="Required field"
    >
      *
    </span>
  </label>
  <div
    className="css-0"
  >
    <div
      aria-expanded={false}
      aria-haspopup="listbox"
      aria-labelledby="downshift-1-label"
      aria-owns={null}
      className="css-0"
      role="combobox"
    >
      <div>
        <div
          className="css-1ky7qjs"
        >
          <input
            aria-activedescendant={null}
            aria-autocomplete="list"
            aria-controls={null}
            autoComplete="off"
            className="css-15ndqf0"
            id="downshift-1-input"
            name="school"
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            placeholder="Divination, Abjuration, Necromancy..."
            required={true}
            type="text"
            value=""
          />
          <span
            className="css-djhpe5"
          >
            <button
              className="css-mwsknu"
              disabled={false}
              onClick={[Function]}
              tabIndex={-1}
              type="button"
            >
              <span>
                <svg
                  aria-hidden="true"
                  className="css-n86my2"
                  height="1em"
                  role="img"
                  viewBox="0 0 24 24"
                  width="1em"
                >
                  <path
                    d="M16.294 8.294a1 1 0 111.414 1.414l-5 5a1 1 0 01-1.414 0l-5.001-5a1 1 0 011.414-1.415l4.294 4.294 4.293-4.293z"
                  />
                </svg>
              </span>
            </button>
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Spell Fields/SpellDuration Default Render 1`] = `
<div
  className="css-0"
>
  <label
    className="css-1tdx2hb"
    htmlFor="duration"
  >
    Duration
    <span
      className="css-fpulsy"
      title="Required field"
    >
      *
    </span>
  </label>
  <div
    className="css-o1mvac"
  >
    <input
      className="css-15ndqf0"
      id="duration"
      name="duration"
      placeholder="Instaneous, 1 minute, 1 hour..."
      required={true}
      type="text"
    />
  </div>
</div>
`;

exports[`Storyshots Spell Fields/SpellRange Default Render 1`] = `
<div
  className="css-0"
>
  <label
    className="css-1tdx2hb"
    htmlFor="range"
  >
    Range
    <span
      className="css-fpulsy"
      title="Required field"
    >
      *
    </span>
  </label>
  <div
    className="css-o1mvac"
  >
    <input
      className="css-15ndqf0"
      id="range"
      name="range"
      placeholder="Self, 30 feet, 120 feet..."
      required={true}
      type="text"
    />
  </div>
</div>
`;

exports[`Storyshots Spell Fields/Type Default Render 1`] = `
<div
  className="css-0"
>
  <label
    className="css-1tdx2hb"
    htmlFor="level_number"
  >
    Type
    <span
      className="css-fpulsy"
      title="Required field"
    >
      *
    </span>
  </label>
  <div
    className="css-0"
  >
    <div
      aria-expanded={false}
      aria-haspopup="listbox"
      aria-labelledby="downshift-2-label"
      aria-owns={null}
      className="css-0"
      role="combobox"
    >
      <div>
        <div
          className="css-1ky7qjs"
        >
          <input
            aria-activedescendant={null}
            aria-autocomplete="list"
            aria-controls={null}
            autoComplete="off"
            className="css-15ndqf0"
            id="downshift-2-input"
            name="level_number"
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            placeholder="Cantrip, 1st level..."
            required={true}
            type="text"
            value=""
          />
          <span
            className="css-djhpe5"
          >
            <button
              className="css-mwsknu"
              disabled={false}
              onClick={[Function]}
              tabIndex={-1}
              type="button"
            >
              <span>
                <svg
                  aria-hidden="true"
                  className="css-n86my2"
                  height="1em"
                  role="img"
                  viewBox="0 0 24 24"
                  width="1em"
                >
                  <path
                    d="M16.294 8.294a1 1 0 111.414 1.414l-5 5a1 1 0 01-1.414 0l-5.001-5a1 1 0 011.414-1.415l4.294 4.294 4.293-4.293z"
                  />
                </svg>
              </span>
            </button>
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Spell card Homebrew Spell 1`] = `
<div
  className="css-1ckdz6j"
  onClick={[Function]}
>
  <div
    className="css-o2430k"
  >
    <div
      className="css-1gl7d4a"
    >
      <button
        className="css-5sb8u8"
        onClick={[Function]}
        type="button"
      >
        <span>
          <svg
            aria-hidden="true"
            className="css-n86my2"
            height="1em"
            role="img"
            viewBox="0 0 24 24"
            width="1em"
          >
            <path
              d="M16.294 8.294a1 1 0 111.414 1.414l-5 5a1 1 0 01-1.414 0l-5.001-5a1 1 0 011.414-1.415l4.294 4.294 4.293-4.293z"
            />
          </svg>
        </span>
      </button>
      <span
        className="css-kjh4ly"
      >
        Blinding Flash
      </span>
    </div>
    <button
      className="css-5sb8u8"
      onClick={[Function]}
      type="button"
    >
      <span>
        <svg
          aria-hidden="true"
          className="css-n86my2"
          height="1em"
          role="img"
          viewBox="0 0 24 24"
          width="1em"
        >
          <defs>
            <path
              d="M2 12a2 2 0 11-.001 4.001A2 2 0 012 12zm0-6a2 2 0 11-.001 4.001A2 2 0 012 6zm0-6a2 2 0 11-.001 4.001A2 2 0 012 0z"
              id="dots_svg__a"
            />
          </defs>
          <use
            transform="translate(10 4)"
            xlinkHref="#dots_svg__a"
          />
        </svg>
      </span>
    </button>
  </div>
  <div
    className="css-15r1rym"
    hidden={true}
  >
    <div
      className="css-j7qwjs"
    >
      <div
        className="css-wstido"
      >
        <p
          className="css-lc0qpe"
        >
          5th level
           
          Illusion
        </p>
      </div>
      <div
        className="css-wstido"
      >
        <div>
          <p
            className="css-1ocwri7"
          >
            <b>
              Casting time:
            </b>
             
            1 action
          </p>
          <p
            className="css-1ocwri7"
          >
            <b>
              Range:
            </b>
             
            30 feet
          </p>
          <p
            className="css-1ocwri7"
          >
            <b>
              Components:
            </b>
             
            S, M
          </p>
          <p
            className="css-1ocwri7"
          >
            <b>
              Duration:
            </b>
             
            1 minute
          </p>
        </div>
      </div>
      <div
        className="css-wstido"
      >
        <div
          className="css-16ewyma"
        >
          <p
            className="css-1ocwri7"
          >
            A creature within range is subjected to a blinding flash before its eyes. The target must make both a Constitution saving throw and a Dexterity saving throw. If the target succeeds on both saves, nothing happens. If the target fails one save but succeeds on the other, it is blinded for the duration. If it fails both saves, it is blinded for the duration, and stunned until the end of your next turn.
          </p>
          <p
            className="css-1ocwri7"
          >
            At the end of each of its turns, a blinded target can make a new Constitution saving throw, ending the effect on a success. It cannot make this saving throw if it is stunned or otherwise incapacitated.
          </p>
          <p
            className="css-1ocwri7"
          >
            At Higher Levels. When you cast this spell using a spell slot of 6th level or higher, you can target one additional creature for each slot level above 5th.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Spell card Srd Spell 1`] = `
<div
  className="css-1ckdz6j"
  onClick={[Function]}
>
  <div
    className="css-o2430k"
  >
    <div
      className="css-1gl7d4a"
    >
      <button
        className="css-5sb8u8"
        onClick={[Function]}
        type="button"
      >
        <span>
          <svg
            aria-hidden="true"
            className="css-n86my2"
            height="1em"
            role="img"
            viewBox="0 0 24 24"
            width="1em"
          >
            <path
              d="M16.294 8.294a1 1 0 111.414 1.414l-5 5a1 1 0 01-1.414 0l-5.001-5a1 1 0 011.414-1.415l4.294 4.294 4.293-4.293z"
            />
          </svg>
        </span>
      </button>
      <span
        className="css-kjh4ly"
      >
        Alter Self
      </span>
      <span
        className="css-1eji9m2"
      >
        SRD
      </span>
    </div>
  </div>
  <div
    className="css-15r1rym"
    hidden={true}
  >
    <div
      className="css-j7qwjs"
    >
      <div
        className="css-wstido"
      >
        <p
          className="css-lc0qpe"
        >
          2nd level
           
          Enchantment
        </p>
      </div>
      <div
        className="css-wstido"
      >
        <div>
          <p
            className="css-1ocwri7"
          >
            <b>
              Casting time:
            </b>
             
            1 action
          </p>
          <p
            className="css-1ocwri7"
          >
            <b>
              Range:
            </b>
             
            Self
          </p>
          <p
            className="css-1ocwri7"
          >
            <b>
              Components:
            </b>
             
            V, S
          </p>
          <p
            className="css-1ocwri7"
          >
            <b>
              Duration:
            </b>
             
            Concentration, up to 1 hour
          </p>
        </div>
      </div>
      <div
        className="css-wstido"
      >
        <div
          className="css-16ewyma"
        >
          <p
            className="css-1ocwri7"
          >
            You assume a different form. When you cast the spell, choose one of the following options, the effects of which last for the duration of the spell. While the spell lasts, you can end one option as an action to gain the benefits of a different one.
          </p>
          <p
            className="css-1ocwri7"
          >
            Aquatic Adaptation: You adapt your body to
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots TestComponent Text 1`] = `
<p>
  Hello world
</p>
`;
